src/patternfly/components/Toolbar/toolbar.scss

```
@@ -6,7 +6,7 @@ $pf-v5-c-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
 $pf-v5-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
 .#{$toolbar} {
-  --#{$toolbar}--AlignItems--base: flex-start;
+  --#{$toolbar}--AlignItems--base: start;
```

This should probably stay as `flex-start` until safari 17, it only got support in 15.4.

```suggestion
  --#{$toolbar}--AlignItems--base: flex-start;
```